scripts/requirements.esp32.txt

```diff
@@ -10,3 +10,6 @@ pygdbmi<=0.9.0.2
 reedsolo>=1.5.3,<=1.5.4
 bitstring>=3.1.6
 ecdsa>=0.16.0
+kconfiglib==13.7.1
+construct==2.10.54
+python-socketio<5
```

Comment:
@mspang Can you please review the scripts related changes?

Reply:
What's the problem with version 5?

Reply:
Hey @mspang I was just referring to the generic changes in requirements.txt for ESP32. Nothing specific :-)

Reply:
If there is nothing specific then please do not use a less-than requirement.

Reply:
As documented there: https://github.com/espressif/esp-idf/blob/v4.3/requirements.txt#L20, there was some compatible issue. We will check with our tools team, and update upstream if the issue has been fixed.

Reply:
Thanks for bug link.